tests/entrypoints/llm/test_lazy_outlines.py

```diff
@@ -36,7 +36,7 @@ def run_lmfe(sample_regex):
     llm = LLM(model="facebook/opt-125m",
               enforce_eager=True,
               guided_decoding_backend="lm-format-enforcer",
-              gpu_memory_utilization=0.6)
```

Comment:
I find this test is not enough. if gc works well, this vllm instance will be destroyed when the function exits. so i added another test.